packages/frontend/src/main/components/viewer/ObjectInfoResource.vue

```diff
@@ -98,38 +97,30 @@ export default {
   },
   computed: {
     isolated() {
-      return this.viewerState.isolateValues.indexOf(this.resource.data.object.id) !== -1
+      if (!this.viewerState.currentFilterState?.isolatedObjects) return false
+
+      return this.viewerState.currentFilterState?.isolatedObjects?.includes(
+        this.resource.data.object.id
+      )
     },
     visible() {
-      return this.viewerState.hideValues.indexOf(this.resource.data.object.id) === -1
+      if (!this.viewerState.currentFilterState?.hiddenObjects) return true
+
+      return !this.viewerState.currentFilterState?.hiddenObjects?.includes(
+        this.resource.data.object.id
+      )
     }
   },
   methods: {
     isolate() {
       const id = this.resource.data.object.id
-      if (this.isolated)
-        unisolateObjects({
-          filterKey: '__parents',
-          filterValues: [id]
-        })
-      else
-        isolateObjects({
-          filterKey: '__parents',
-          filterValues: [id]
-        })
+      if (this.isolated) unIsolateObjects([id], 'ui-res', true)
```

Comment:
ui-res - another special value that could be a constant? if all of these values have special meaning for Viewer, maybe the Viewer package should export them out. if not, they could still be constants somewhere else, because they're repeated multiple times